mopidy_spotify/translator.py

```diff
@@ -103,14 +103,14 @@ def to_track(sp_track, bitrate=None):
         return
 
     artists = [to_artist(sp_artist) for sp_artist in sp_track.artists]
-    artists = filter(None, artists)
+    artists = [a for a in artists if a]
 
     album = to_album(sp_track.album)
 
     return models.Track(
         uri=sp_track.link.uri,
         name=sp_track.name,
-        artists=artists,
+        artists=list(artists),
```

Comment:
I guess `list()` was needed because `filter()` returned an iterator. Now it should be an actual list, so the `list()` call here is probably not needed anymore.